modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -271,22 +275,21 @@ The record payloads for random access are exposed as an `immutable List`, and si
 
 === Batch Aggregator Characteristics
 
-* The batch aggregator scope can only exist in batch steps which, in turn, are only usable within the batch process phase of a batch job. You cannot use batch aggregators during the On Complete phase of a batch job.
-* An aggregator can only wrap the final element within the batch step in which it resides.
-* Several *Anypoint Connectors* can handle record-level errors without failing a whole batch aggregation (i.e., upsert). +
+* The Batch Aggregator scope can only exist in a Batch Step which, in turn, is only usable within the batch process phase of a Batch Job. You cannot use Batch Aggregator components during the On Complete phase of a Batch Job.
```

Comment:
what's the logic for the read flow of this page? Why does it go from filtering to batch aggregator, then to block size and finally back to aggregators?